benchmark/dune

```diff
@@ -0,0 +1,4 @@
+(executable
```

Comment:
Currently, this setup is still in a monorepo even though it won't be built when run from top-level. Dune doesn't build executables if they're not needed, but if you `esy dune build @all` or `esy x something` from the project root, this will be compiled (and fail, because root doesn't know about core_bench).

Make the root `dune` file to look something like:

```
(env
 (_ (flags (:standard -w -9))))
(dirs :standard \ benchmark)
```

Reply:
I'm assuming you want to run benchmarks like:

```
cd benchmark
esy
esy run
```